pkg/sslpolicy.go

```diff
@@ -13,10 +13,10 @@ import (
 // necessary to create our globally enforced SSL Policy.
 func NewSslPolicy(name string) compute.SslPolicy {
 	return compute.SslPolicy{
-		Description:   "Commercetools TLS policy: modern features and TLS 1.2 only.",
+		Description:   fmt.Printf("Commercetools TLS policy: %s features and above TLS %s only.", config.SslProfile(), config.TlsVersion()),
```

Comment:
please remove the company name here

Reply:
It also doesn't need Printf in here

Reply:
@namm2 it's being there since the beginning, just updated to make it conpatible with the changes

Reply:
It looks like the Printf is added later (green) than it's there since the beginning, see:
![image](https://user-images.githubusercontent.com/46129063/80058295-31919800-8529-11ea-9863-63e140096143.png)

Reply:
fmt.Printf might be needed for string interpolation here.

Reply:
Yes, that's the point. Thanks for your explanation.